src/features/winterStorageApplicationList/WinterStorageApplicationListContainer.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useQuery } from '@apollo/react-hooks';
+
+import WinterStorageApplicationList from './WinterStorageApplicationList';
+import InternalLink from '../../common/internalLink/InternalLink';
+import Table, { Column, COLUMN_WIDTH } from '../../common/table/Table';
+import { formatDate } from '../../common/utils/format';
+import Chip from '../../common/chip/Chip';
+import { APPLICATION_STATUS } from '../../common/utils/consonants';
+import { ApplicationStatus } from '../../@types/__generated__/globalTypes';
+import { getWinterStorageApplicationData, WinterStorageApplication } from './utils';
+import { usePagination } from '../../common/utils/usePagination';
+import { useBackendSorting } from '../../common/utils/useBackendSorting';
+import {
+  WINTER_STORAGE_APPLICATIONS,
+  WINTER_STORAGE_APPLICATIONSVariables as WINTER_STORAGE_APPLICATIONS_VARS,
+} from './__generated__/WINTER_STORAGE_APPLICATIONS';
+import { WINTER_STORAGE_APPLICATIONS_QUERY } from './queries';
+import Pagination from '../../common/pagination/Pagination';
+import TableFilters from '../../common/tableFilters/TableFilters';
+import ApplicationDetails from '../../common/applicationDetails/ApplicationDetails';
+
+type ColumnType = Column<WinterStorageApplication>;
+
+const WinterStorageApplicationListContainer = () => {
+  const { t, i18n } = useTranslation();
+  const { cursor, pageSize, pageIndex, getPageCount, goToPage } = usePagination();
+
+  const { orderBy, handleSortedColChange } = useBackendSorting(() => goToPage(0));
+  const { loading, data } = useQuery<WINTER_STORAGE_APPLICATIONS, WINTER_STORAGE_APPLICATIONS_VARS>(
+    WINTER_STORAGE_APPLICATIONS_QUERY,
+    {
+      variables: {
+        first: pageSize,
+        after: cursor,
+        orderBy,
+      },
+    }
+  );
+  const columns: ColumnType[] = [
+    {
+      Cell: ({ cell }) => (
+        <InternalLink to={`/winter-storage-applications/${cell.value}`}>
+          {t('applicationList.applicationType.newApplication')}
+        </InternalLink>
+      ),
+      Header: t('applicationList.tableHeaders.applicationType') || '',
+      accessor: 'id',
+      filter: 'exact',
+      disableSortBy: true,
+      width: COLUMN_WIDTH.M,
+    },
+    {
+      Cell: ({ cell }) => formatDate(cell.value, i18n.language),
+      Header: t('applicationList.tableHeaders.pvm') || '',
+      accessor: 'createdAt',
+      width: COLUMN_WIDTH.S,
+    },
+    {
+      Header: t('applicationList.tableHeaders.queue') || '',
+      accessor: 'queue',
+      disableSortBy: true,
+      width: COLUMN_WIDTH.XS,
+    },
+    {
+      Header: t('applicationList.tableHeaders.municipality') || '',
+      accessor: 'municipality',
+      disableSortBy: true,
+      width: COLUMN_WIDTH.S,
```

Comment:
The first letter should be in uppercase, though the data might be the issue here.

Reply:
Noticed this as well, but I don't think we should include capitalization logic here as the lower case comes from the data.